documentation/quarkus/getting-started-quarkus.asciidoc

```diff
@@ -0,0 +1,234 @@
+= Quarkus Quickstart 
+
+This guide will show a quickstart how to create a Quarkus app and will briefly show the key functionalities that Quarkus provides and how to start as a beginner or also experienced developers.
+
+
+== Introduction to Quarkus 
+
+To get the first introduction to Quarkus you can read the link:../quarkus.asciidoc[Quarkus introduction guide]. That guide contains a chapter **Guide to the Reader** for experienced Spring developers that have already followed devon4j guidelines that shows a brief summary of the important devon4j guidelines and recommendations. 
```

Comment:
The "Guide to the Reader" is not only for experienced Spring developers. It can be a guide for anyone who wants to know where to find information when they come in contact with Quarkus